### PR TITLE
upm readiness

### PR DESCRIPTION
**Pull request by NicholasPerell**
Getting the files arranged so we have the ability to submit this to OpenUPM if we decide to

### CODE REVIEW

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+github: VirtualMaker
+patreon: RageAgainstThePixel
```
Comment (StephenHodgson) [inline .github/FUNDING.yml:2]:
```suggestion
```

Comment (StephenHodgson) [reply .github/FUNDING.yml:2]: We could probably get rid of this file altogether

File: .github/workflows/upm-subtree-split.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+jobs:
+  upm-subtree-split:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+    - name: upm subtree split
+      run: |
```
Comment (StephenHodgson) [inline .github/workflows/upm-subtree-split.yml:18]: I have an action for this.

Comment (StephenHodgson) [reply .github/workflows/upm-subtree-split.yml:18]: https://github.com/RageAgainstThePixel/upm-subtree-split

File: Bindings/Packages/com.virtualmaker.bindings/package.json [hunk 3]
```diff
@@ -1,8 +1,28 @@
...
+    "documentationUrl": "https://github.com/virtual-maker-net/com.virtualmaker.bindings#documentation",
+    "changelogUrl": "https://github.com/virtual-maker-net/com.virtualmaker.bindings/releases",
+    "license": "MIT",
+    "repository": {
+        "type": "git",
+        "repository": "https://github.com/virtual-maker-net/com.virtualmaker.bindings.git"
+    },
+    "author": {
+        "name": "VirtualMaker",
+        "url": "https://www.virtualmaker.dev/"
```
Comment (StephenHodgson) [inline Bindings/Packages/com.virtualmaker.bindings/package.json:18]: Is this URL correct? 

I thought it was .net

Comment (NicholasPerell) [reply Bindings/Packages/com.virtualmaker.bindings/package.json:18]: .dev is our studio; .net is the product

File: .github/workflows/upm-subtree-split.yml (hunk 2 above)
Comment (NicholasPerell) [reply .github/workflows/upm-subtree-split.yml]: Think I've added it correctly, now?

File: .github/FUNDING.yml (hunk 1 above)
Comment (NicholasPerell) [reply .github/FUNDING.yml]: Removed 🫡

File: .github/workflows/upm-subtree-split.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    - main
+jobs:
+  upm-subtree-split:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+    - name: UPM subtree split
+      uses: RageAgainstThePixel/upm-subtree-split@v1.1.3
```
Comment (StephenHodgson) [inline .github/workflows/upm-subtree-split.yml:15]:
```suggestion
    - uses: RageAgainstThePixel/upm-subtree-split@v1.1.3
```

File: .github/workflows/copy-documentation-readme-upwards.yml
Comment (StephenHodgson) [inline .github/workflows/copy-documentation-readme-upwards.yml:1]: This fine might have broken references to image links if not properly checked.

File: Bindings/Packages/com.virtualmaker.bindings/Documentation~/README.md [hunk 5]
```diff
@@ -1,41 +1,177 @@
-# com.virtualmaker.bindings
+# VirtualMaker Bindings
 
-[![Discord](https://img.shields.io/discord/855294214065487932.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/xQgMW9ufN4) [![openupm](https://img.shields.io/npm/v/com.virtualmaker.bindings?label=openupm&registry_uri=https://package.openupm.com)](https://openupm.com/packages/com.virtualmaker.bindings/) [![openupm](https://img.shields.io/badge/dynamic/json?color=brightgreen&label=downloads&query=%24.downloads&suffix=%2Fmonth&url=https%3A%2F%2Fpackage.openupm.com%2Fdownloads%2Fpoint%2Flast-month%2Fcom.virtualmaker.bindings)](https://openupm.com/packages/com.virtualmaker.bindings/)
```
Comment (StephenHodgson) [inline Bindings/Packages/com.virtualmaker.bindings/Documentation~/README.md]: lets keep all our social tags

File: Bindings/Packages/com.virtualmaker.bindings/Documentation~/README.md [hunk 6]
```diff
@@ -1,41 +1,177 @@
-# com.virtualmaker.bindings
+# VirtualMaker Bindings
```
Comment (StephenHodgson) [inline Bindings/Packages/com.virtualmaker.bindings/Documentation~/README.md]: nit: revert, should be the name of the package

**Review by StephenHodgson — DISMISSED** [review]
Bring back the social badges and installation steps

File: Bindings/Assets/Tests/UxmlTests.cs [hunk 7]
```diff
@@ -51,7 +51,7 @@ public void IntegerFieldTests()
     public void TextFieldTests()
     {
         _stringProperty.Value = "Hello";
-        _bindings.BindTextField("textField", _stringProperty, true);
+        _bindings.BindField("textField", _stringProperty, true);
```
Comment (StephenHodgson) [inline Bindings/Assets/Tests/UxmlTests.cs:54]: should be split into different PR?

**Comment by StephenHodgson** [comment]
Some files like .gitignore should be in root, not subdir of unity project.
